### PR TITLE
Added new wording for unattended installer as well as new arch

### PR DESCRIPTION
**Pull request by enclave-tom**
relates to #74

### CODE REVIEW

File: docs/setup/windows.md [hunk 1]
```diff
@@ -71,91 +71,145 @@ If your installation fails and you are unable to resolve the problem, please con
...
+        Providing an invalid enrolment key will cause the installation to fail (and roll back).
 
-To update an unattended installation, distribute the new version of the unattended installer via the same method as you used when deploying originally; existing enrolled systems will have their configuration and identity preserved during the upgrade.
+        !!! note
+              If your tooling provides an enrolment key to the installer, at the time of installation the target system will require an Internet connection to perform the enrolment.
+              
+              If you need to deploy Enclave to a system without an Internet connection, use the unattended installer without the `ENROLMENT_KEY` property, then use the `enclave enrol` CLI command to enrol the system when an Internet connection becomes available.
 
-The running instance of Enclave will stop and then restart with the new version automatically.
+=== "32-bit executable"
```
Comment (enclave-alistair) [inline docs/setup/windows.md:119]: We should add an equivalent block for our arm64 executable.

File: docs/setup/windows.md [hunk 2]
```diff
@@ -71,91 +71,145 @@ If your installation fails and you are unable to resolve the problem, please con
...
+        </div>
 
-Removing Enclave from a system (when Enclave was installed via unattended mode) can either be done by normal Windows uninstall mechanisms (i.e. via Apps and Features), or by running the unattended installer with the `-uninstall` argument. For example, to perform a silent uninstall:
+    2. Distribute as needed via Group Policy, Configuration Manager, or any other deployment method that allows installers to run with elevated permissions on the machine.
 
-### Distributing Enclave as an MSI file
+        To run the installer in silent mode (with no UI), you pass the `-q` command line argument, like so:
+
+        ```
+        ./enclave-setup-unattended-x32-<version>.exe -q
```
Comment (enclave-alistair) [inline docs/setup/windows.md:136]: I think the file name is `x86`?

File: docs/setup/windows.md [hunk 3]
```diff
@@ -71,91 +71,145 @@ If your installation fails and you are unable to resolve the problem, please con
...
+
+### Uninstalling
+
+Removing Enclave from a system (when Enclave was installed via unattended mode) can either be done by normal Windows uninstall mechanisms (i.e. via Apps and Features), or by running the unattended installer with the `-uninstall` argument.
+
+
+
+## Troublehsooting install issues
+
+If an installion doesn't work as expected, examine the log files at `C:\Program Files\Enclave Networks\Enclave\Agent\logs`. If an install is interrupted, use the Enclave setup file to restart the process.
```
Comment (enclave-alistair) [inline docs/setup/windows.md:188]: Typo; `installion` -> `installation`.

File: docs/setup/windows.md [hunk 4]
```diff
@@ -71,91 +71,145 @@ If your installation fails and you are unable to resolve the problem, please con
...
+
+To update an unattended installation, distribute the new version of the unattended installer via the same method as you used when deploying originally; existing enrolled systems will have their configuration and identity preserved during the upgrade. Enclave will stop and then restart with the new version automatically during the update.
+
+### Uninstalling
+
+Removing Enclave from a system (when Enclave was installed via unattended mode) can either be done by normal Windows uninstall mechanisms (i.e. via Apps and Features), or by running the unattended installer with the `-uninstall` argument.
+
+
+
+## Troublehsooting install issues
```
Comment (enclave-alistair) [inline docs/setup/windows.md:186]:
```suggestion
## Troubleshooting install issues
```

**Review by enclave-alistair — CHANGES_REQUESTED** [review]

File: docs/setup/windows.md [hunk 5]
```diff
@@ -71,91 +71,145 @@ If your installation fails and you are unable to resolve the problem, please con
...
+        ```
+
+        Providing an invalid enrolment key will cause the installation to fail (and roll back).
+
+        !!! note
+              If your tooling provides an enrolment key to the installer, at the time of installation the target system will require an Internet connection to perform the enrolment.
+              
+              If you need to deploy Enclave to a system without an Internet connection, use the unattended installer without the `ENROLMENT_KEY` property, then use the `enclave enrol` CLI command to enrol the system when an Internet connection becomes available.
+
+=== "64-bit MSI package"
```
Comment (enclave-alistair) [inline docs/setup/windows.md:153]: I'm guessing we should probably add the x86 and arm64 MSI package links as well?

Comment (enclave-tom) [reply docs/setup/windows.md:153]: It's there already?

Comment (enclave-alistair) [reply docs/setup/windows.md:153]: I can see the exe ones but not the MSI ones?

**Review by enclave-alistair — APPROVED** [review]
👍

**Pull request merged**